### PR TITLE
Initial package configuration, basic structure

### PR DESCRIPTION
**Pull request by lslezak**
- I'm not merging to `master` but to a separate branch so we can do the review in smaller parts and merge to `master` when ready to submit the first version.
- The files were mostly copied and adapted from `yast2-migration`
- More will follow

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    $ git clone https://github.com/yast/<repository>.git
+
+If you want to contribute into the project you can
+[fork](https://help.github.com/articles/fork-a-repo/) the repository and clone your fork.
+
+
+Development Environment
+=======================
+
+The module is developed only for openSUSE.
```
Comment (dgdavid) [inline README.md:45]: only for migrating openSUSE to SLES ?

File: package/yast2-migration-sle.spec [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+Requires:       yast2-registration
+Requires:       yast2-migration
+
+Supplements:    yast2-registration
+
+BuildArch: noarch
+
+%description
+This package contains the YaST2 component for migrating an openSUSE
+system to the SUSE Linux Enterprise (SLE) system.
```
Comment (dgdavid) [inline package/yast2-migration-sle.spec:47]: More specifically to SLES, right?

**Review by dgdavid — APPROVED** [review]
In general looks good to me, but I'm a little confused with references to SLE and SLES. Please, check them to see if they can be improved or simply continue leaving as they are.

File: package/yast2-migration-sle.spec (hunk 2 above)
Comment (lslezak) [reply package/yast2-migration-sle.spec:47]: Um, currently yes, but in theory you could migrate to SLE4SAP, SLES-HPC,... it just depends what SCC returns.

File: README.md (hunk 1 above)
Comment (lslezak) [reply README.md:45]: Yes, currently openSUSE Leap -> SLES is the only supported case.

**Pull request merged**